### DISCUSSION TITLE
Does dragonflydb support unix socket connection like redis?

### DISCUSSION THREAD

**Discussion by hnimminh** [dragonflydb/dragonfly#168] [Q&A] [answered]
Interesting project, Does dragonflydb support unix socket connection like Redis?

**Comment by romange** [comment, marked as answer]
No, it does not. Why do you need uds?

**Reply by hnimminh** [reply]
usecase of that is just use dragonflydb as the local cache db, unix has reduce the network call (RTT) as well and many other advantage over TCP/IP network call

**Reply by romange** [reply]
I am less familiar with uses-cases where Redis is used locally. If you think this is something you must have in order to use Dragonfly, feel free to open a feature request and explain your case.

**Reply by romange** [reply]
Specifically, I wonder why Redis is not good enough as local cache db.

**Reply by hnimminh** [reply]
just a simple reason dragonflydb is redis replacement and dragonflydb is really good, why not have that :)
